joomla/joomla-cms
# [4.0] Optimize Discover and Update menu badges

### Summary of Changes

Run simplified query for `Discover` and `Update` badges on System Dashboard.

### Testing Instructions

Drop some extension files in your Joomla! installation so they can be discovered.
Install some outdated extension.
Check that `Discover` and `Update` badge still work correctly.

### Expected result

Works like before.

### Documentation Changes Required

No.
